platform/viewer/src/studylist/StudyListWithData.js

```diff
@@ -32,6 +32,10 @@ class StudyListWithData extends Component {
 
   static defaultProps = {
     studyListFunctionsEnabled: true,
+    server:
+      window.config.enableGoogleCloudAdapter && window.config.oidc[0].server
```

Comment:
It's a bit dangerous to pull from `window.config` here, as it may not be set (null).

These values should be available in the new `appContext` created in: https://github.com/OHIF/Viewers/pull/852

Also, I'm not sure this is where I would set a default value. Maybe in `mounted` if it's not defined here and is set in `appContext`? That would also allow us to set this route prop by modifying the URL (later on). Which I believe is the heart of what this related issue is requesting: https://github.com/OHIF/Viewers/issues/753